apis/projectcontour/v1alpha1/contourconfig.go

```diff
@@ -120,6 +120,10 @@ type GlobalCircuitBreakerDefaults struct {
 	// The maximum number of parallel retries a single Envoy instance allows to the Kubernetes Service; defaults to 3.
 	// +optional
 	MaxRetries uint32 `json:"maxRetries,omitempty" yaml:"max-retries,omitempty"`
+
+	// PerHostMaxConnections is the maximum number of connections
+	// that Envoy will allow to each individual host in a cluster.
+	PerHostMaxConnections uint32 `json:"perHostMaxConnections,omitempty" yaml:"per-host-max-connections,omitempty"`
```

Comment:
lets add to the release note that this is configurable on the global settings now as well since it wasn't here before

Reply:
done